install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -1,5 +1,30 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
+metadata:
+  name: machine-config-controller
+  namespace: openshift-machine-config-operator
+  labels:
+    k8s-app: machine-config-controller
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  groups:
+    - name: mcc-important-config-paosed
+      rules:
+        - alert: MCCImportantConfigPaused
+          expr: |
+             mcc_important_config_paused > 0
+          labels:
+            severity: warning
+          annotations:
+            message: "Important config {{$labels.desc}} ({{ $labels.file }}) is blocked by pause in {{ $labels.pool }}"
```

Comment:
I'm pretty sure you want `summary` and `description`, but not `message`.  For example, see openshift/cluster-version-operator#547, which was successfully verified by QE, and [these monitoring docs][1], which call for `summary` and `description`, but do not mention the older `message`.

[1]: https://github.com/monitoring-mixins/docs/pull/10/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R183